<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="fontShow"/>
     </item>
     <item>
      <widget class="QPushButton" name="changeFont">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Behaviours</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="antiAlias">
            <property name="text">
             <string>Antialiased font</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="whiteSpace">
            <property name="text">
             <string>Visualize whitespaces</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="hightLine">
            <property name="text">
             <string>Hightlight current line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showEndl">
            <property name="text">
             <string>Visualize end of line</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Theme</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addTheme">
          <property name="text">
           <string>Add theme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delTheme">
          <property name="text">
           <string>Delete theme</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QListView" name="listView"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="ColorSelect" name="backColor" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorSelect" name="foreColor" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="bold">
            <property name="text">
             <string>Bold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="italic">
            <property name="text">
             <string>Italic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="underline">
            <property name="text">
             <string>Underline</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="strike">
            <property name="text">
             <string>Strike</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorSelect</class>
   <extends>QWidget</extends>
   <header>includes/color-select.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
